### PR TITLE
PXB-3307 Release notes and release note tasks for 8.0.35-31

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch pxb-3307
	modified:   docs/release-notes.md
	new file:   docs/release-notes/8.0/8.0.35-31.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: When XtraBackup performs a backup and exports a tablespace for tables that have changed their charset-collation before the backup, it creates an incorrect table definition. This incorrect table definition means the table's structure information may not match the data.
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.35-31.0.md]: If backup contains tables with temporal datatypes (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate correct configuration (.cfg)files required by the import.

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* [PXB-2797]:  When you import a table with a full-text index from an XtraBackup backup, you might encounter a schema mismatch. This happens when the table's structure in the backup does not match the structure expected by the database.
+
+* [PXB-3210]: Percona XtraBackup version 8.0.33-28 and later fail to build on MacOS because of the FIND_PROCPS() function. This function causes issues during the build process, preventing the software from compiling successfully.
+
+* [PXB-3130]: During an upgrade, if a type is out of bounds, the Prepare step fails. This means that a value exceeds the expected range, causing the upgrade process to stop and not complete.
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.35-31.0.md]: when preparing 8.0.30 backups with newer xtrabackup versions, prepare step failed.

Comment (satya-bodapati) [reply docs/release-notes/8.0/8.0.35-31.0.md]: And please add thank you note to the contributor LiNing / githubid: @LI8023NING

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

**Comment by satya-bodapati** [comment]
@patrickbirch we should also add a note that PXB 8.0.35 release (starting from 8.0.34) is upward compatible. People are still unware about this change

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Bug fix
+
+* [PXB-3283]: If backup contains tables with [temporal datatypes] (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate the correct configuration (.cfg)files required by the import.
+
+* [PXB-2797]:  When you import a table with a full-text index from an XtraBackup backup, you might encounter a schema mismatch. This happens when the table's structure in the backup does not match the structure expected by the database.
+
+* [PXB-3210]: Percona XtraBackup version 8.0.33-28 and later fail to build on MacOS because of the FIND_PROCPS() function. This function causes issues during the build process, preventing the software from compiling successfully.
+
+* [PXB-3130]: when preparing 8.0.30 backups with newer xtrabackup versions, The Prepare step failed. (Thank you to LiNing who contributed)
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.35-31.0.md]: P in Prepare can be lower case?
The/the?
Thank you LiNing for the contribution!

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Check the latest list of all [Supported versions](..//..//supported-versions.md).
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: If backup contains tables with [temporal datatypes] (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate the correct configuration (.cfg)files required by the import.
```
Comment (satya-bodapati) [inline docs/release-notes/8.0/8.0.35-31.0.md]: space between (.cfg) and files

Comment (satya-bodapati) [reply docs/release-notes/8.0/8.0.35-31.0.md]: couldn't prepare vs didn't prepare, please choose the better version, I don't know ;)

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
minor changes requested

**Review by satya-bodapati — APPROVED** [review]
LGTM!

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: If backup contains tables with [temporal datatypes] (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate the correct configuration (.cfg) files required by the import.
+
+* [PXB-2797]:  When you import a table with a full-text index from an XtraBackup backup, you might encounter a schema mismatch. This happens when the table's structure in the backup does not match the structure expected by the database.
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0/8.0.35-31.0.md]: [PXB-2797]:  When -> remove extra space

File: docs/release-notes/8.0/8.0.35-31.0.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+* [PXB-3210]: Percona XtraBackup version 8.0.33-28 and later fail to build on MacOS because of the FIND_PROCPS() function. This function causes issues during the build process, preventing the software from compiling successfully.
+
+* [PXB-3130]: when preparing 8.0.30 backups with newer xtrabackup versions, The prepare step failed. (Thank you to LiNing for the contribution!)
```
Comment (alina-derkach-oaza) [inline docs/release-notes/8.0/8.0.35-31.0.md]: when preparing -> When preparing
The prepare step failed - the prepare step failed
(Thank you to LiNing for the contribution!) -> (Thanks to LiNing for the contribution!)

File: docs/release-notes/8.0/8.0.35-31.0.md (hunk 5 above)
Comment (patrickbirch) [reply docs/release-notes/8.0/8.0.35-31.0.md]: Two "when" words in the sentence.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**